File: python/liczby23.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  liczby23.py


def liczby2():
    """
    Funkcja drukuje wszystkie liczby 2- crfrowe, w którch nie powatrzają się cyfry. 
    Na koniec zwraca ilośc takich liczb.
    Wykluczone liczby: 11,,22,33 itd.
    
    """
    
    
    licznik = 0
    for i in range(1, 10): # Pętla dziesiątek
        for j in range(10):# Petla jedności
            if j != i:                                   #"!=" oznacza jest rózna
                print("{}{} ".format(i, j), end="")
                licznik = licznik + 1 #zliczanie liczb
    return licznik



def liczby3():
    """
    Funkcja drukuje wszystkie liczby 2- crfrowe, w którch nie powatrzają się cyfry. 
    Na koniec zwraca ilośc takich liczb.
    Wykluczone liczby: 11,,22,33 itd.
    
    """
    
    
    licznik = 0
    for i in range(1, 10):        # Pętla dziesiątek
        for j in range(10):          # Petla jedności
            for k in range(10):
                if i != j != k != i:                                   #"!=" oznacza jest rózna
                    print("{}{}{} ".format(i, j, k), end=" ")
                    licznik = licznik + 1 #zliczanie liczb
    return licznik







def main(args):
    
    
     print("\n\nliczczb 2-cyfrowych:", liczby2())
     print(" ")
     print("\n\nliczczb 3-cyfrowych:", liczby3())
    
    
    
    
    
    
    

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: cpp/liczby23.cpp
/*
 * liczby23.cpp

 */



#include <iostream>

using namespace std;


int liczby2(){
    int ile = 0;
    for(int i = 1; i < 10;i++) {
        for(int j = 1; j <10; j++) {
            
            if (j != i) {
                    cout << i << j << " ";
                    ile++;
                }
            }
        }  
        return ile;
}
    


int liczby3(){
    int ile = 0;
    for(int i = 1; i < 10;i++) {
        for(int j = 1; j < 10; j++) {
			for(int k = 1; k < 10; k++){
            
            if (j != i != k != i) {
                    cout << i << j << k << " ";
                    ile++;
                }
            }
        }  
        
    }

        return ile;
	}

    













int main(int argc, char **argv)
{

	cout << "liczby 2 cyfrowe" << liczby2();
    cout << "\n\ " ;
    cout << "liczby 2 cyfrowe" << liczby3();

    
	return 0;
}

